mesa/visualization/ModularVisualization.py

```diff
@@ -236,25 +287,38 @@ class ModularServer(tornado.web.Application):
     # Handlers and other globals:
     page_handler = (r"/", PageHandler)
     socket_handler = (r"/ws", SocketHandler)
+    health_handler = (
```

Comment:
When/how is this being used?

Reply:
It's the check path on Kubernetes clusters so you can tell when the container is ready for service.

Reply:
Again as per commit message

Reply:
Sorry, didn't think of reading the commit message :see_no_evil:
In the line below is "healthz" intentional (the z) or by accident

Reply:
sorry, still hadnt read teh commit message, seems intentional

Reply:
'healthz' is the default path for Kubernetes health checks. It annoys me as well.